src/mudlet.pro

```diff
@@ -1339,4 +1331,8 @@ DISTFILES += \
     mudlet-lua/tests/README.md \
     mudlet-lua/tests/DB.lua \
     mudlet-lua/tests/GUIUtils.lua \
-    mudlet-lua/tests/Other.lua
+    mudlet-lua/tests/Other.lua \
+    ../mudlet.desktop \
```

Comment:
This is of consequence on FreeBSD as well as Linux - so having it to hand in Qt Creator is useful - IMHO.